prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -61,26 +61,25 @@
 
 // WR - work request
 #define FI_IBV_RDM_SERVICE_WR_MASK              ((uint64_t)0x1)
-#define FI_IBV_RDM_CHECK_SERVICE_WR_FLAG(value) \
+#define FI_IBV_RDM_CHECK_SERVICE_WR_FLAG(value)	\
         (value & FI_IBV_RDM_SERVICE_WR_MASK)
 
-#define FI_IBV_RDM_PACK_WR(value)               ((uint64_t)value)
-#define FI_IBV_RDM_UNPACK_WR(value)             ((void*)(uintptr_t)value)
+#define FI_IBV_RDM_PACK_WR(value)               ((uint64_t)(uintptr_t)(void *)value)
+#define FI_IBV_RDM_UNPACK_WR(value)             ((void *)(uintptr_t)value)
 
-#define FI_IBV_RDM_PACK_SERVICE_WR(value)       \
-        (((uint64_t)(uintptr_t)(void*)value) | FI_IBV_RDM_SERVICE_WR_MASK)
+#define FI_IBV_RDM_PACK_SERVICE_WR(value)					\
+        (((uint64_t)(uintptr_t)(void *)value) | FI_IBV_RDM_SERVICE_WR_MASK)
 
-#define FI_IBV_RDM_UNPACK_SERVICE_WR(value)     \
-        ((void*)(uintptr_t)(value & (~(FI_IBV_RDM_SERVICE_WR_MASK))))
+#define FI_IBV_RDM_UNPACK_SERVICE_WR(value)				\
+        ((void *)(uintptr_t)(value & (~(FI_IBV_RDM_SERVICE_WR_MASK))))
 
 // Send/Recv counters control
 
-#define FI_IBV_RDM_INC_SIG_POST_COUNTERS(_connection, _ep, _send_flags)		\
+#define FI_IBV_RDM_INC_SIG_POST_COUNTERS(_connection, _ep)			\
 do {                                                                		\
 	int32_t sends_outgoing =						\
 		ofi_atomic_inc32(&(_connection)->av_entry->sends_outgoing);	\
 	ofi_atomic_inc32(&(_ep)->posted_sends);					\
-	(_send_flags) |= IBV_SEND_SIGNALED;                             	\
```

Comment:
Hmm.. if all post sends are not signaled now, when the counters are decremented? And how does rndv work?

Reply:
This patch does `qp_init_attr::sq_sig_all = 1`. It guarantees that all posted send operations will be completed with an appropriate WC.